.eslintrc.js

```diff
@@ -9,6 +9,6 @@ module.exports = {
     "function-paren-newline": "off",
     "no-restricted-syntax": "off",
     "react/no-deprecated": "off",
-    "import/no-anonymous-default-export": "off",
```

Comment:
This lint rule is responsible for throwing an error on `export default { ... }` since it is an object literal.